v2_1/ws/rest/src/sdmx-rest.yaml

```diff
@@ -152,7 +168,9 @@ paths:
 
   /metadata/metadataflow/{agencyID}/{resourceID}/{version}/{providerID}:
```

Comment:
If metadata attributes are allowed being multi-valued and thus restricting the occurrence of a metadata attribute at any hierarchy level to max. 1, would allow adding an additional parameter containing the metadata attribute:  
/metadata/metadataflow/{agencyID}/{resourceID}/{version}/{providerID}/{metadataAttrID}

Reply:
Thanks, @dosse. I don't have a view on this but the scope of this pull request is different and of a much more modest scope, i.e. it's merely about adding tags to the queries so as to categorize them :).